### DISCUSSION TITLE
Vaultwarden Alpine script updates??

### DISCUSSION THREAD

**Discussion by niekniek89** [tteck/Proxmox#3693] [General] [answered]
Hello,

I just used the script to deploy a vaultwarden alpine machine.

**bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/alpine-vaultwarden.sh)"**

However, it keeps saying there is an update available, while 1.32.0 is the most recent (see screenshot).
Am I doing something wrong?
Am I maybe in a beta branch or something?
The screenshot is from the admin page.

![image](https://github.com/user-attachments/assets/8fc2c0f4-3d36-4ad0-8a3f-c87f12317960)

**Comment by tteck** [comment, marked as answer]
Please learn how to use the search feature.
https://github.com/tteck/Proxmox/discussions?discussions_q=alpine+vaultwarden+

**Reply by niekniek89** [reply]
Thanks, I had already looked around there by chance.
However (for me) I have not found a clear answer yet.
I will look again soon.

**Reply by niekniek89** [reply]
I've gone through a few, but don't understand the branch parts.
Do I need to change anything, or do I have the latest releases by default now?
